src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
These imports are not used in the `FruitTransaction` class. Remember to remove any unused imports to keep the code clean.